### PR TITLE
#4983 Remote image load error handling

### LINKED ISSUE #4983: re-enable viewing remote images in signed/encrypted messages still not working as expected.
Showing remote images still don't work well yet.

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR implemented remote image load error handling

close #4983 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Review by sosnovsky — COMMENTED** [review]
Let's add some border around image block, as currently it looks the same as plain text:

Screenshot 2023-03-06 at 20 19 33

Here I added `div` around error block with such style:
```css
border: 1px solid #d5d5d5; 
display: inline-block; 
padding: 10px 20px; 
text-align: center; 
line-height: 1.8; 
margin: 12px 0;
font-size: 0.9em;
color: #666;
```

File: extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts [hunk 1]
```diff
@@ -328,6 +328,9 @@ export class PgpBlockViewRenderModule {
           image.addEventListener('load', () => this.resizePgpBlockFrame());
         }
       } catch (e) {
+        // When CORS error, set error message to `Remote server blocked image loading`
+        const errorMessage = e.message?.includes('Failed to fetch') ? 'Remote server blocked image loading' : 'Failed to load image';
```
Comment (sosnovsky) [inline extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts:332]: Firefox currently shows `Failed to load image` for CORS error, as it has different `e.message` - `NetworkError when attempting to fetch resource`

Comment (sosnovsky) [reply extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts:332]: Ah, it seems `Failed to fetch` and `NetworkError when attempting to fetch resource` errors are shown not only for CORS errors, but for other cases too, like bad internet connection or some network error.

Then let's show `Failed to load image` for all errors

Comment (ioanmo226) [reply extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts:332]: Sounds good

**Review by sosnovsky — COMMENTED** [review]
Text centering looks strange:
Screenshot 2023-03-07 at 11 13 41

And let's make it smaller, so it doesn't take too much space.
For this we can use just `<br>` instead of `<p>`, like this:
```html
Failed to load image<br>
<a href="https://test-site.com/bad-image" target="_blank">Open image in a new tab</a>
```

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**